File: Python/SoftUni PB/Exercises/Python Basics/Conditional Statements - Exercise/Lunch_break.py
import math

series = input()
series_duration = int(input())
break_duration = int(input())

lunch_duration = 1/8 * break_duration
time_for_rest = 1/4 * break_duration

remaining_time_for_series = break_duration - \
                            lunch_duration - time_for_rest
needed_time = abs(remaining_time_for_series - series_duration)
if remaining_time_for_series >= series_duration:
    print(f"You have enough time to watch {series} and left with {math.ceil(needed_time)} "
          f"minutes free time.")
else:
    print(f"You don't have enough time to watch {series}, you need {math.ceil(needed_time)} more minutes.")

File: JS-Basics/Conditional Statements/Conditional Statements - Exercise/lunchBreak.js
function lunchBreak(input) {
    let seriesName = input[0];
    let timePerEpisode = Number(input[1]);
    let breakTime = Number(input[2]);

    let lunchBreak = breakTime / 8;
    let relaxTime = breakTime / 4;
    let freeTime = breakTime - (lunchBreak + relaxTime);

    if (freeTime > timePerEpisode || freeTime == timePerEpisode) {
        remainingTime = freeTime - timePerEpisode;
        console.log(`You have enough time to watch ${seriesName} and left with ${Math.ceil(remainingTime)} minutes free time.`);
    } else {
        remainingTime = timePerEpisode - freeTime;
        console.log(`You don't have enough time to watch ${seriesName}, you need ${Math.ceil(remainingTime)} more minutes.`);
    }
}

lunchBreak(["Teen Wolf","48","60"]);
